spring-cloud-gcp-data-spanner/src/test/java/com/google/cloud/spring/data/spanner/core/admin/SpannerDatabaseAdminTemplateTests.java

```diff
@@ -64,7 +64,7 @@
 
 	private List<String> ddlList;
 
-	@Before
+	@BeforeEach
 	public void setup() {
```

Comment:
FYI: AFAIK, all `@Test` (and perhaps this one too) can be `private` now.

Reply:
Yeah I studied that in the official JUnit5 guide but I wasn't so sure about it because I couldn't find a good technical reason to change the methods visibility. 
They've suggested to keep the methods public by default because of two reasons, one being the possibility of a test class being extended by test class in another package and the other to simplify testing on the module path when using the Java Module System. 
Please let me know what you think about this, I'd love to have a new perspective about it.

Reply:
Package private is the default I've seen most commonly -- it's less visible (and less verbose!) than `public`, but it still lets other classes in the same package extend.

Reply:
I'll have a look if we have any extensions of classes/methods outside packages in the code. If we don't on a majority, it might make sense to go classes/methods being package-private, right ?

Reply:
Yes, let's stick to package-private default for everything where it does not cause a problem.

Reply:
@ddixit14 By the way, if you take a look at the code smells (Sonar check), you'll see that it advises to remove "public" modifier. I like reading Sonar's "why is this an issue" explanations -- most of the time I agree; some of the time I don't, but I almost always learn something.

Reply:
@chanseokoh for example, I just learned (had no idea!) that JUnit5 runner will [silently ignore private methods](https://sonarcloud.io/organizations/googlecloudplatform/rules?open=java%3AS5810&rule_key=java%3AS5810).